crates/forge/tests/e2e/running.rs

```diff
@@ -614,6 +614,7 @@ fn init_new_project_test() {
             [package]
             name = "test_name"
             version = "0.1.0"
+            edition = "2023_10"
```

Comment:
Is this now a required parameter?

Reply:
It's optional, if not filled in it defaults to an edition set as default on compiler side - currently `2023_01`

Reply:
If not specified it uses the default, but this is the expected toml content. We use `scarb new` which generates it in the Scarb.toml of the new package